code/modules/admin/topic.dm

```diff
@@ -71,6 +71,11 @@
 				log_admin("[key_name(usr)] created a emergency response team.")
 				if(!src.makeEmergencyresponseteam())
 					usr << "<span class='danger'>Unfortunatly there were not enough candidates available.</span>"
+			if("14")
+				message_admins("[key_name(usr)] created an ERP squad.")
+				log_admin("[key_name(usr)] created an ERP squad.")
+				if(!src.makeERPsquad())
+					usr << "<span class='danger'>Fortunatly there were not enough candidates available.</span>"

```

Comment:
Hey don't blame me for this one, the guy above me I copied it from fucked it up too